custom/ilsgateway/api.py

```diff
@@ -105,13 +106,13 @@ def __init__(self, base_uri, username, password):
 
     def get_supplypointstatuses(self, domain, facility, **kwargs):
         meta, supplypointstatuses = self.get_objects(self.supplypointstatuses_url, **kwargs)
-        location_id = _get_location_id(facility, domain)
+        location_id = SQLLocation.objects.get(domain=domain, external_id=facility).location_id
```

Comment:
I changed this because supplypointstatus is also generated for non reporting types.